Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "vrf_dalek"
+name = "cardano_vrf_rs"
 version = "0.1.0"
-authors = ["iquerejeta <querejeta.azurmendi@iohk.io>"]
+authors = ["iquerejeta <querejeta.azurmendi@iohk.io>", "hal@cardanofoundation.org", "paweljakubas"]
```

Comment:
@abailly any suggestions here? Maybe something connected with amaru? Pallas?

Reply:
unfortunately we do not have differentiation between authors and maintainers in manifest : https://doc.rust-lang.org/cargo/reference/manifest.html